platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/exceptions/MapboxConfigurationException.java

```diff
@@ -17,7 +17,7 @@
    * Creates a Mapbox configuration exception thrown by MapboxMap when the SDK hasn't been properly initialised.
    */
   public MapboxConfigurationException() {
-    super("\nUsing MapView requires calling Mapbox.getInstance(Context context, String accessToken) before "
+    super("\nUsing MapView requires calling Mapbox.initialize(Context context, String accessToken) before "
```

Comment:
This should still point to `Mapbox.getInstance`.

Reply:
refactor issue, nice catch